metadata/identifier/subscribe_metadata_identifier.go

```diff
@@ -17,17 +17,18 @@
 
 package identifier
 
+// SubscriberMetadataIdentifier is inherit baseMetaIdentifier with service params: Revision
 type SubscriberMetadataIdentifier struct {
 	revision string
 	BaseMetadataIdentifier
 }
 
-// getIdentifierKey...
+// GetIdentifierKey will return string format as service:Version:Group:Side:Revision
```

Comment:
getIdentifierKey's return string format is "service:Version:Group:Side:Revision"